### PR TITLE
created dashboard oc config (deployment, service, path, imagestream)

### PR DESCRIPTION
**Pull request by sakalosj**

### CODE REVIEW

File: openshift/dashboard.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+spec:
+  tags:
+    - name: stg
+      from:
+        kind: DockerImage
+        name: usercont/packit-dashboard:{{ deployment }}
+      importPolicy:
+        # Periodically query registry to synchronize tag and image metadata.
+        # DOES NOT WORK on Openshift Online.
+        scheduled: true
```
Comment (jpopelka) [inline openshift/dashboard.yml]: I changed this in #45

File: playbooks/deploy.yml [hunk 2]
```diff
@@ -87,6 +87,15 @@
...
 
+    - name: Deploy dashboard (no need to process)
+      # https://docs.ansible.com/k8s_module.html
+      k8s:
+        namespace: "{{ project }}"
+        definition:  "{{ lookup('template', '../openshift/dashboard.yml')  }}"
+        host: "{{ host }}"
+        api_key: "{{ api_key }}"
+        verify_ssl: "{{ validate_certs }}" # 2.7 compatibility
+
```
Comment (jpopelka) [inline playbooks/deploy.yml]: We could perhaps deploy it conditionally, see for example
* https://github.com/packit-service/deployment/blob/master/vars/template.yml#L46
* https://github.com/packit-service/deployment/blob/master/playbooks/deploy.yml#L110

File: openshift/dashboard.yml (hunk 1 above)
Comment (sakalosj) [reply openshift/dashboard.yml]: updated

File: playbooks/deploy.yml (hunk 2 above)
Comment (sakalosj) [reply playbooks/deploy.yml]: updated

**Comment by jpopelka** [comment]
Ok, you most likely don't have the [pre-commit](https://pre-commit.com) installed.
See [instructions](https://github.com/packit-service/deployment/blob/master/.pre-commit-config.yaml#L3) and install it in all cloned repos which contain `.pre-commit-config.yaml` (e.g. `ogr`, `packit`, `packit-service`).

You'll need to amend your commit (`git commit --amend -a`) to trigger it this time (next time it'll be triggered with each commit automatically).
Don't panic when one of the checks/formaters fail, just check (`git diff`) what it's formated and repeat the amend command and everything will be fine, I promise :-)

**Comment by sakalosj** [comment]
> Ok, you most likely don't have the [pre-commit](https://pre-commit.com) installed.
> See [instructions](https://github.com/packit-service/deployment/blob/master/.pre-commit-config.yaml#L3) and install it in all cloned repos which contain `.pre-commit-config.yaml` (e.g. `ogr`, `packit`, `packit-service`).
> 
> You'll need to amend your commit (`git commit --amend -a`) to trigger it this time (next time it'll be triggered with each commit automatically).
> Don't panic when one of the checks/formaters fail, just check (`git diff`) what it's formated and repeat the amend command and everything will be fine, I promise :-)

fixed now, please check

File: openshift/dashboard.yml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        - name: packit-dashboard
+          image: usercont/packit-dashboard:{{ deployment }}
+          ports:
+            - containerPort: 8443
+          volumeMounts:
+            - name: packit-secrets
+              mountPath: /secrets
+          env:
+            - name: DEPLOYMENT
+              value: { { deployment } }
```
Comment (jpopelka) [inline openshift/dashboard.yml]:
```suggestion
              value: {{ deployment }}
```

File: openshift/dashboard.yml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+spec:
+  tags:
+    - name: stg
+      from:
+        kind: DockerImage
+        name: usercont/packit-dashboard:{{ deployment }}
+      importPolicy:
+        # Periodically query registry to synchronize tag and image metadata.
+        # DOES NOT WORK on Openshift Online.
+        scheduled: { { auto_import_images } }
```
Comment (jpopelka) [inline openshift/dashboard.yml]:
```suggestion
        scheduled: {{ auto_import_images }}
```

Comment (sakalosj) [reply openshift/dashboard.yml]: fixed

File: openshift/dashboard.yml (hunk 3 above)
Comment (sakalosj) [reply openshift/dashboard.yml]: fixed

File: openshift/dashboard.yml [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: packit-dashboard
+spec:
+  tags:
+    - name: stg
+      from:
+        kind: DockerImage
+        name: usercont/packit-dashboard:{{ deployment }}
```
Comment (jpopelka) [inline openshift/dashboard.yml]: If we wanted to be consistent with other images (image streams) then we'd used a variable here, let's say:
```suggestion
        name: {{ image_dashboard }}
```
which would be defined in `playbooks/deploy.yml` as
```yaml
    image_dashboard: "usercont/packit-dashboard:{{ deployment }}"
```

See also #55

Comment (sakalosj) [reply openshift/dashboard.yml]: fixed
i realized already and  am totally ok with it :)

**Pull request merged**